### PR TITLE
Add UX design to 2FA settings page [SCI-4814]

### PR DESCRIPTION
**Pull request by aignatov-bio**
Jira ticket: [SCI-4814](https://biosistemika.atlassian.net/browse/SCI-4814)

### What was done
Add UX design to 2FA settings page

### CODE REVIEW

File: config/initializers/constants.rb [hunk 1]
```diff
@@ -205,6 +205,21 @@ class Constants
 
   ACADEMY_BL_LINK = 'https://scinote.net/academy/?utm_source=SciNote%20software%20BL&utm_medium=SciNote%20software%20BL'.freeze
 
+  TWO_FACTOR_URL = {
+    google: {
+      android: 'https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2',
+      ios: 'https://apps.apple.com/us/app/google-authenticator/id388497605'
```
Comment (okriuchykhin) [inline config/initializers/constants.rb:211]: Is it account specific URL?

Comment (radiokills) [reply config/initializers/constants.rb:211]: no, it's the id of the app

Comment (radiokills) [reply config/initializers/constants.rb:211]: https://apps.apple.com/us/app/alex-is-the-king/id388497605

File: app/assets/stylesheets/themes/scinote.scss [hunk 2]
```diff
@@ -469,8 +469,10 @@ a[data-toggle="tooltip"] {
 }
 
 .user-statistics {
+  margin-top: 1em;
+
   .list-inline {
-    margin-left: 15px;
+    margin-left: 0;
```
Comment (urbanrotnik) [inline app/assets/stylesheets/themes/scinote.scss:475]: Can we remove this? Or we override some default value with 0?

File: app/views/users/registrations/edit_partials/_password.html.erb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      </div>
+
+      <div class="form-group sci-input-container">
+        <%= f.label :password_confirmation, t("users.registrations.edit.new_password_2_label") %>
+        <%= f.password_field :password_confirmation, autocomplete: "off", class: "form-control sci-input-field", "data-role" => "clear" %>
+      </div>
+
+      <div class="align-right">
+        <a href="#" class="btn btn-light" data-action="cancel"><%=t "general.cancel" %></a>
+        <%= f.submit t("general.save"), class: "btn btn-success" %>
```
Comment (urbanrotnik) [inline app/views/users/registrations/edit_partials/_password.html.erb:36]: f.button?

File: app/assets/stylesheets/themes/scinote.scss (hunk 2 above)
Comment (aignatov-bio) [reply app/assets/stylesheets/themes/scinote.scss:475]: We override with 0

File: app/views/users/registrations/edit_partials/_password.html.erb (hunk 3 above)
Comment (aignatov-bio) [reply app/views/users/registrations/edit_partials/_password.html.erb:36]: I just copy to partial

**Pull request merged**